cmd/influxd/server_integration_test.go

```diff
@@ -433,9 +476,9 @@ func runTestsData(t *testing.T, testName string, nodes Cluster, database, retent
 			expected: `{"results":[{"series":[{"name":"logs","columns":["time","event"]}]}]}`,
 		},
 		{
-			name:     "missing measurement with `GROUP BY *`",
-			query:    `select load from "%DB%"."%RP%".missing group by *`,
-			expected: `{"results":[{"error":"measurement not found: \"%DB%\".\"%RP%\".\"missing\""}]}`,
+			name:          "missing measurement with `GROUP BY *`",
+			query:         `select load from "%DB%"."%RP%".missing group by *`,
+			expectPattern: `.*measurement not found: .*missing.*`,
```

Comment:
Oh, we should definitely include the measurement name, database, etc. It's just the source line I find strange. For it to mean anything I may have to go back to a tag in the repo, find the source, look at what's going on etc. Many users -- operators for example -- will not care about the source. I agree that "what part of the code" couldn't find the, say, measurement is important, but that might better served with something like "measurement foo not found during field lookup", or whatever.